Shared/Extensions/JellyfinAPI/BaseItemDto/BaseItemDto.swift

```diff
@@ -264,6 +264,28 @@ extension BaseItemDto {
         originalTitle != displayTitle ? originalTitle : nil
     }
 
+    /// Can this `BaseItemDto` be played
+    var presentPlayButton: Bool {
+        switch type {
+        case .aggregateFolder, .basePluginFolder, .boxSet, .folder, .genre, .musicGenre, .person, .photo, .photoAlbum, .studio,
```

Comment:
It would be easier to grep these values if they were flipped: listing values explicitly that are true instead.